pandas/core/dtypes/cast.py

```diff
@@ -5,8 +5,8 @@
 from __future__ import annotations
 
 from datetime import (
-    datetime,
-    timedelta,
+    datetime as datetime_func,
+    timedelta as timedelta_func,
 )
```

Comment:
would `import datetime as dt` (and then `dt.datetime`, `dt.timedelta`) work?

Reply:
sounds good.